### PR TITLE
Adding Event JSON for Accessibility

### PR DESCRIPTION
**Pull request by JRicha024**

### CODE REVIEW

File: cli/examples/accessibility-onClosedCaptionsSettings1.event.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (michaelcfine) [inline cli/examples/accessibility-onClosedCaptionsSettings1.event.json:1]: Please change the name of this file to match the method name... e.g., add "Changed" to the file name.

File: cli/examples/accessibility-onClosedCaptionsSettings2.event.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (michaelcfine) [inline cli/examples/accessibility-onClosedCaptionsSettings2.event.json:1]: Please change the name of this file to match the method name... e.g., add "Changed" to the file name.

**Review by michaelcfine — CHANGES_REQUESTED** [review]
See comments... file renaming, that's all.

File: cli/examples/accessibility-onClosedCaptionsSettings1.event.json (hunk 1 above)
Comment (JRicha024) [reply cli/examples/accessibility-onClosedCaptionsSettings1.event.json]: done

File: cli/examples/accessibility-onClosedCaptionsSettings2.event.json (hunk 2 above)
Comment (JRicha024) [reply cli/examples/accessibility-onClosedCaptionsSettings2.event.json]: done

**Review by michaelcfine — APPROVED** [review]
👍

**Pull request merged**